internal/cmd/asyncapi/account_details.go

```diff
@@ -32,7 +32,8 @@ type channelDetails struct {
 }
 
 type accountDetails struct {
-	cluster        *ccstructs.KafkaCluster
+	clusterId      string
+	kafkaRest      *cmd.KafkaREST
```

Comment:
Why does `accountDetails` contain `kafkaRest`? Is `kafkaRest` and account detail?

Reply:
Usually this would belong in the `command` struct.